2018/en/schedule.md

```diff
@@ -3,62 +3,48 @@ title: Schedule
 layout: default-en-2018
 ---
 
-## Friday (1<sup>st</sup> December 2017, internal)
+## Friday (30<sup>th</sup> November 2018, internal)
```

Comment:
remove internal. its internal in any case and don't raises false hopes.

Reply:
Will do, thanks